web/src/modules/public/alert/sagas/handleAlertSaga.ts

```diff
@@ -66,6 +66,12 @@ export function* handleAlertSaga(action: AlertPush) {
                     yield call(callAlertData, action);
                 }
                 break;
+            case 429:
+                if (action.payload.message.indexOf('identity.session.rack_attack_limit') > -1) {
+                    yield call(callAlertData, action);
+                }
+
+                return;
```

Comment:
the action `callAlertData` is called by default, no need to move that to separate case